cli/lib/exec/spawn.js

```diff
@@ -137,7 +137,15 @@ module.exports = {
 
         const child = cp.spawn(executable, electronArgs, stdioOptions)
 
-        child.on('close', resolve)
+        function resolver (event) {
```

Comment:
since this is a curried function it would read better as a verb... maybe `resolveWith`...

```
child.on('close', resolveWith('close'))
child.on('exit', resolveWith('exit'))

Reply:
or `resolveOn('close')` and `resolveOn('exit')`

Reply:
fair, updated to resolveOn